### PR TITLE
#157645105 get a users request

### PR DESCRIPTION
**Pull request by TMDav007**
#### What does this PR do?
- This PR GET a users request from the  dummy endpoint

#### Description of Task to be completed?
-  The task completed enables a user get a request from the api

#### Any background context you want to provide?
- This task was completed using the Nodejs/Express framework web

#### What are the relevant Github Issues?
- None

#### Questions:
- None

### CODE REVIEW

File: server/dummyControllers/requestController.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import requests from './../dummyModels/request';
+import users from './../dummyModels/user';
 import dummyControllerFunction from './controllerFunction';
```
Comment (nebanat) [inline server/dummyControllers/requestController.js]: default exports should match the base file name e.g. **import SomeFunction from '../../SomeFunction.js'**

File: server/dummyControllers/requestController.js [hunk 2]
```diff
@@ -36,6 +37,44 @@ class RequestController {
...
   }
+
+  /**
+ * it GET all requests
+ * @param {string} req
+ * @param {string} res
+ * @returns {object} object
+ */
+  static getARequest(req, res) {
+    const id = parseInt(req.params.id, 10);
```
Comment (nebanat) [inline server/dummyControllers/requestController.js:47]: check for edge cases do you what happens when you parseInt a string?

File: server/dummyControllers/requestController.js [hunk 3]
```diff
@@ -36,6 +37,44 @@ class RequestController {
...
+
+    // if request is not found
+    if (newRequest.length === 0) {
+      return res.status(404).json({
+        status: 'error',
+        message: 'request not found'
+      });
+    }
+    // get user in users db
+    const foundUser = users.some(user => user.name === newRequest[0].name);
```
Comment (nebanat) [inline server/dummyControllers/requestController.js]: This function returns a request I am wondering why you are returning users in the same function. Always abide by the SRP principle

File: server/dummyControllers/requestController.js (hunk 1 above)
Comment (TMDav007) [reply server/dummyControllers/requestController.js]: ok

File: server/dummyControllers/requestController.js (hunk 2 above)
Comment (TMDav007) [reply server/dummyControllers/requestController.js:47]: ok

File: server/dummyControllers/requestController.js (hunk 3 above)
Comment (TMDav007) [reply server/dummyControllers/requestController.js]: ok

File: server/dummyControllers/requestController.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
-import requests from './../dummyModels/request';
+import requests from './../dummyModels/requests';
 import dummyControllerFunction from './controllerFunction';
```
Comment (nebanat) [inline server/dummyControllers/requestController.js]: default export does not match base file name...

**Pull request merged**